catris/views.py

```diff
@@ -540,6 +553,13 @@ def __init__(self, client: Client, game: Game, player: Player):
         # no idea why these need explicit type annotations
         self.game: Game = game
         self.player: Player = player
+        self._paused_menu = _Menu(
+            [("Continue playing", game.toggle_pause), ("Quit game", self._quit_game)],
```

Comment:
Presumably the first menu item should always be selected when pausing the game, even if you selected "Quit game" last time.